eo-runtime/src/main/java/org/eolang/AtOnce.java

```diff
@@ -78,4 +80,17 @@ public void put(final Phi phi) {
             )
         );
     }
+
+    @Override
+    public void put(final int pos, final Phi phi) {
+        throw new ExReadOnly(
+            String.format(
+                "There are no void attributes left, can't set the %s one",
+                new RuleBasedNumberFormat(
```

Comment:
@Suban05 I believe we don't need to use it here, we can implement this number formatting by our own